test/Adapter/Driver/Sqlsrv/SqlSrvIntegrationTest.php

```diff
@@ -50,4 +50,22 @@ public function testCreateStatement()
         $this->setExpectedException('Zend\Db\Adapter\Exception\InvalidArgumentException', 'only accepts an SQL string or a Sqlsrv resource');
         $driver->createStatement(new \stdClass);
     }
+
+    public function testParameterizedQuery()
+    {
+        $driver = new Sqlsrv([]);
+
+        $resource = sqlsrv_connect(
```

Comment:
This test is exactly the same as the previous one, but with a different driver instance: this should be abstracted in a base test, or moved to an existing base test